system-container/tmpfiles.template

```diff
@@ -0,0 +1,2 @@
+d    $VAR_LIB_OPENSHIFT_INSTALLER - - - - -
```

Comment:
That is the template configuration file for systemd-tmpfiles.

Variable is defined in https://github.com/openshift/openshift-ansible/pull/3720/files#diff-0152a7b8552088b3d310df6cfb5d6b76R133